java/runtime/src/main/java/org/ray/runtime/RayMultiWorkerNativeRuntime.java

```diff
@@ -65,6 +65,7 @@ public RayMultiWorkerNativeRuntime(RayConfig rayConfig, FunctionManager function
       threads[i] = new Thread(() -> {
         RayNativeRuntime runtime = new RayNativeRuntime(rayConfig, functionManager);
         runtimes[workerIndex] = runtime;
+        MultiRuntimeHelper.put(runtime.getWorkerContext().getCurrentWorkerId(), runtime.getTaskExecutor());
```

Comment:
is it possible to put this into the thread local runtime?

Reply:
Because of the native thread pool, it seems hard to put it into thread local runtime.
This helper is used to cache the worker id -> runtime/executor.